xarray/core/dataset.py

```diff
@@ -2030,20 +2033,30 @@ def head(
         Dataset.thin
         DataArray.head
         """
+        if not indexers_kwargs:
+            if indexers is None:
+                indexers = 5
+            if not isinstance(indexers, int) and not is_dict_like(indexers):
```

Comment:
Should we do anything with negative numbers?  (Galaxy brain is to call the opposite head / tail method, but I think that's too clever)

Reply:
I think we should raise an error.